WebHostLib/check.py

```diff
@@ -30,7 +31,15 @@ def check():
                 flash(options)
             else:
                 results, _ = roll_options(options)
-                return render_template("checkResult.html", results=results)
+                if len(options) > 1:
```

Comment:
shouldn't this be `len(files)` to not offer download for a single yaml file with `---` already in it?

Reply:
I tried uploading a combined yaml and it doesn't offer a download.

Reply:
oh, i see. options is actually individual files, not individual options. somehow i didn't expect that